src/network/core/tcp_coordinator.h

```diff
@@ -125,6 +126,7 @@ class NetworkCoordinatorSocketHandler : public NetworkTCPSocketHandler {
 	 *  uint8   Game Coordinator protocol version.
 	 *  uint8   Game-info version used by this client.
 	 *  string  Revision of the client.
+	 *  uint32  Last known listing version, when Game Coordinator procotol >= 4.
```

Comment:
Suggestion:
`(Game Coordinator protocol >= 4) Cursor as received from GC_NEWGRF_LOOKUP, or zero.`

`when` is a bit weird in this context, as it is unclear what it references. But this field doesn't exist before 4, so making that a bit more clear might help.